plugins/snowflake/dbt/include/snowflake/macros/adapters.sql

```diff
@@ -1,14 +1,28 @@
 {% macro snowflake__create_table_as(temporary, relation, sql) -%}
   {%- set transient = config.get('transient', default=true) -%}
+  {# TODO: Add support for more than one keys #}
+  {%- set cluster_by_keys = config.get('cluster_by') -%}
 
   create or replace {% if temporary -%}
     temporary
   {%- elif transient -%}
     transient
   {%- endif %} table {{ relation }}
   as (
-    {{ sql }}
+    {%- if cluster_by_keys is not none -%}
+      select * from( {{ sql }} ) order by ({{ cluster_by_keys }})
```

Comment:
We've been bitten by a bug where the user's last line of code starts with a line comment, eg:
```
select ..
from ...
-- limit 100
```

When we interpolate user-supplied code into a jinja template, we should always put the "wrapping" code on the line after the user-supplied SQL. As is, this code would render out to:
```
select * from (select ..
from ...
-- limit 100) order by cluster_key_1
```

which would be invalid! So, pretty minor edge case, but this is one where the extra couple of linebreaks are totally worth it!

Reply:
Oh nasty! Didn't think of that. Easy to fix hehe